### ISSUE TITLE
Sonoma installer USB resulting in a reboot loop?

### ISSUE THREAD

**Issue by xtreme1690** [seven-of-eleven/Lenovo-ThinkPad-L13-Yoga-Hackintosh#30]
Hey!

As always, excellent work on the EFI here and glad to hear we can get Sonoma going on our L13Y's.
However i seem to be having a weird issue. So, i've created a Sonoma USB as per Apple's own instructions for making a USB installer. Then i've grabbed the latest EFI from here to get it bootable on the system.

The only changes i've made to the EFI are the SMBIOS (serial/etc), to match what i had with my Monterey install, and enabled the AirportitlwmS.kext and disabled the rest (of the Airport ones that is).

The USB does appear to boot, however just as the display brightness turns itself up the screen goes blank and the whole system restarts. When trying to boot again there is indication a kernel panic occurred as i get the "A problem occurred and your computer was restarted" message when i try to boot the USB again.

So i edited the EFI boot args to enable verbose boot, but i see no indication of a kernel panic at all, the system just appears to be booting the USB and then just as the installer graphical interface would startup the screen goes off and the whole system restarts. I have provided a screenshot just on the off chance something is there that could indicate whats going on? This is the last frame of verbose boot before screen goes blank and system restarts.
![Photo 27-06-2024, 12 39 16](https://github.com/seven-of-eleven/Lenovo-ThinkPad-L13-Yoga-Hackintosh/assets/10789662/b4beb92c-7269-43fb-81ea-72967ae36ca1)

EDIT: Booting into Monterey off of this same new EFI (with the older version of the Airport kext loaded for it) has let me see the kernel panic log, i have attached it as well. Some sort of memory stack corruption going on? 
![Photo 27-06-2024, 12 54 31](https://github.com/seven-of-eleven/Lenovo-ThinkPad-L13-Yoga-Hackintosh/assets/10789662/5be87b78-90c9-46e6-b655-91e215f7497e)

Looks like it has something to do with Airport?

**Comment by bennals** [comment]
I'm seeing the exact same thing, though I didn't have the problem installing Ventura .  My wifi card is an AX201, so different to the type seven-of-eleven has.  What type do you have?

**Comment by bennals** [comment]
I was able to succesfully install Sonoma by swapping the airportitlwmS.kext in this EFI with the release targeting Mac OS 14.4.0 and later from the [OpenIntelWireless repositry](https://github.com/OpenIntelWireless/itlwm):

https://github.com/OpenIntelWireless/itlwm/releases/download/v2.3.0/AirportItlwm_v2.3.0_stable_Sonoma14.4.kext.zip

Once I got past the kernel panic issue I encountered the installer reboot issue that appeared with the release of 14.4, but it's easily fixed as documented [here](https://elitemacx86.com/threads/how-to-fix-reboot-loop-when-installing-or-updating-to-macos-sonoma-14-4.1720/) as well as other places.

After a few hours of using this laptop, it appears to be 100% rock solid and probably the best hackintosh I've ever had.  No issues with touchscreen or stylus pen, though I've only done a little testing in a drawing context with the stylus.

Thanks again for this awesome repo.
